core/string/translation_server.h

```diff
@@ -52,10 +52,14 @@ class TranslationServer : public Object {
 
 	static TranslationServer *singleton;
 	bool _load_translations(const String &p_from);
-	String _standardize_locale(const String &p_locale, bool p_add_defaults) const;
```

Comment:
This should actually have been part of #98743. I can split that out to a separate PR if that's preferred.